troupon/templates/snippet_deal_thumbnail.html

```diff
@@ -1,9 +1,20 @@
 {% load humanize %}
 
 <div class="grid-item card">
+  <form class="overlay row" action="/cart/view/" method="POST">
+    {% csrf_token %}
+    <input type="hidden" name="slug" value="{{deal.slug}}" />
```

Comment:
**Code Quality: Nitpick**

I would prefer you made use of the deal id instead of the slug for this.